### PR TITLE
Add switch between DataStore and DataObject in replace menu

### LINKED ISSUE #1372: Add DataStoreReference to DataObjectReference replace menu and vice-versa
__Is your feature request related to a problem? Please describe.__

In its current state of the DataObjectReference replace menu only contains the collection marker.
It feels empty and out of place.
image

As discussed with @andreasgeier, it has been suggested to make it possible for a DataObjectReference to be switched to a DataStoreReference, and vice-versa.
This will ensure that the menu is always populated with something, at least until the DataObjectInputs/Outputs are handled.

### PR DESCRIPTION
**Pull request by azeghers**
__Which issue does this PR address?__

Closes #1372

__Acceptance Criteria__

* [ ] Corresponds to the concept 
* [ ] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
Are the package-lock.json changes intended?

File: lib/features/replace/BpmnReplace.js [hunk 1]
```diff
@@ -247,6 +253,15 @@ export default function BpmnReplace(
       newElement.host = target.host;
     }
 
+    // The DataStoreReference element is 14px wider than the DataObjectReference element
+    // This ensures that they stay centered on the x axis when replaced
+    if (
+      newElement.type === 'bpmn:DataStoreReference' ||
+      newElement.type === 'bpmn:DataObjectReference'
+    ) {
+      newElement.x = element.x + (element.width - newElement.width) / 2;
```
Comment (pinussilvestrus) [inline lib/features/replace/BpmnReplace.js:262]: ❤️

File: lib/features/replace/BpmnReplace.js [hunk 2]
```diff
@@ -247,6 +253,15 @@ export default function BpmnReplace(
...
+    }
+
```
Comment (pinussilvestrus) [inline lib/features/replace/BpmnReplace.js:264]: ❤️

**Comment by azeghers** [comment]
> Are the package-lock.json changes intended?

They aren't, but that's what happens when I `npm install`

Do you have insights on this?

**Comment by MaxTru** [comment]
> > Are the package-lock.json changes intended?
> 
> They aren't, but that's what happens when I `npm install`
> 
> Do you have insights on this?

See https://stackoverflow.com/questions/64813775/is-there-any-way-to-fix-package-lock-json-lockfileversion-so-npm-uses-a-specific. 
Apparently @azeghers uses a different npm version that what was used for bpmn-js initially. (Alexis, didnt we see did already when we looked at another use-case?). Therefore the package-lock.json is recreated with a new `lockfileVersion`.

Therefore I would not include the `package-lock.json` in this case.

**Comment by nikku** [comment]
Please use `npm@6` if possible. Given that version no changes in the `package-lock` should happen on `npm install`. 

@azeghers Can you verify this?

**Comment by azeghers** [comment]
I fought a bit with nvm but I should be on the correct npm version now.
Feel free to re-review!

**Review by pinussilvestrus — APPROVED** [review]

**Pull request merged**